app/styles/helpers/inverse-element.less

```diff
@@ -23,13 +41,17 @@
   .upf-select,
   .selectize-input {
     color: @color-text-contrast;
-    background-color: @upf-primary-rock-blue;
-    border-color: @upf-primary-rock-blue;
+    background-color: fade(white, 10%);
+    border: 0;
 
     &:focus, &:hover {
       outline: none;
       box-shadow: none;
-      background: @upf-primary-rock-blue;
+      background: @inverse-color;
```

Comment:
`background-color` instead of `background`. I myself forgot to change it ... (CSS shorthand rules are Evil)